geth/transactions/queue/queue.go

```diff
@@ -36,33 +33,38 @@ var (
 	ErrInvalidCompleteTxSender = errors.New("transaction can only be completed by the same account which created it")
 )
 
+// remove from queue on any error (except for transient ones) and propagate
+var transientErrs = map[string]bool{
+	keystore.ErrDecrypt.Error():          true, // wrong password
+	ErrInvalidCompleteTxSender.Error():   true, // completing tx create from another account
+	account.ErrNoAccountSelected.Error(): true, // account not selected
+}
+
+type empty struct{}
```

Comment:
I am not sure if this is needed.

Reply:
it was a common practice in some opensource projects to use such type for such cases, but I can avoid using it

Reply:
Let's leave it then. Maybe it's just me and everyone are ok with it.